salt/modules/x509_v2.py

```diff
@@ -255,8 +255,8 @@ def create_certificate(
         Instead of returning the certificate, write it to this file path.
 
     overwrite
-        If ``path`` is specified and the file exists, do not overwrite it.
-        Defaults to false.
+        If ``path`` is specified and the file exists, overwrite it.
+        Defaults to true.
```

Comment:
```suggestion
        If ``path`` is specified and the file exists, overwrite it.
        Defaults to true.
```
Why switch the default behavior?

Reply:
This just fixes documentation (no logic change), which is at least confusing or even incorrect. It's `def create_certificate(..., overwrite=True)` and the code

```
if path and not overwrite and __salt__["file.file_exists"](path):
    return f"The file at {path} exists and overwrite was set to false"
```

So, by default, a file would be overwritten (this was the default behavior in the previous x509 modules as well). Thus "[...] do not overwrite [...] defaults to false" is inverted and suggests setting `overwrite` to true to protect the file.

Reply:
Yeah, my bad, I didn't check the actual function.